app/controllers/users_controller.rb

```diff
@@ -11,7 +11,12 @@ def show
   end
 
   def new
-    @user = User.new
+    if logged_in?
+        flash[:danger] = "You are unauthorized to create new users."
```

Comment:
Hey @ncbui I think a better message is "Already logged in. Sign out first to create a new account." 😉

Reply:
Thanks @msarit, updating the message